tests/plugins/wysiwygarea/manual/detachedembededimagesource.html

```diff
@@ -0,0 +1,26 @@
+<p>Google logo:</p>
+
+<pre>https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png</pre>
```

Comment:
I'm not sure if we should use some resource that we don't have any control over.